backend/spec/features/admin/users_spec.rb

```diff
@@ -152,6 +152,31 @@
       expect(user_a.reload.bill_address.address1).to eq "1313 Mockingbird Ln"
     end
 
+    context 'invalid entry' do
+      around do |example|
+        ::AlwaysInvalidUser = Class.new(Spree.user_class) do
+          validate :always_invalid_email
+          def always_invalid_email
+            errors.add(:email, "is invalid")
+          end
+        end
+        orig_class = Spree.user_class
+        Spree.user_class = "AlwaysInvalidUser"
+
+        example.run
+
+        Spree.user_class = orig_class.name
+        Object.send(:remove_const, "AlwaysInvalidUser")
```

Comment:
yep, rspec around already takes care of this for you. rspec docs are so awful, but here's a demo, you can run this file by itself to see. https://gist.github.com/jrochkind/04725b9f864d0cdedd7c8a7474cdd183